### PR TITLE
Added tests for dropTable, truncateTable and renameTable

### PR DESCRIPTION
**Pull request by edgardmessias**

### CODE REVIEW

File: tests/framework/db/CommandTest.php [hunk 1]
```diff
@@ -347,24 +347,51 @@ public function testAlterTable()
...
     }
 
     public function testRenameTable()
     {
+        $db = $this->getConnection();
+
+        $fromTableName = 'type';
+        $toTableName = 'new_type';
+
+        $db->createCommand('DROP TABLE IF EXISTS {{' . $toTableName . '}};')->execute();
```
Comment (samdark) [inline tests/framework/db/CommandTest.php]: There's no `IF EXISTS` in DBs other than MySQL.

Comment (samdark) [inline tests/framework/db/CommandTest.php]: Better to do it via checking schema and doing drop table via API.

File: tests/framework/db/mssql/MssqlCommandTest.php [hunk 2]
```diff
@@ -127,4 +127,27 @@ public function testAlterTable()
...
+
+    public function testRenameTable()
+    {
+        $db = $this->getConnection();
+
+        $fromTableName = 'type';
+        $toTableName = 'new_type';
+
+        // on the first run the 'new_type' table does not exist
+        try {
```
Comment (samdark) [inline tests/framework/db/mssql/MssqlCommandTest.php]: This approach would work in MySQL well. Let's use it instead of duplicating tests.

**Comment by edgardmessias** [comment]
Fixed @samdark

**Comment by samdark** [comment]
https://github.com/yiisoft/yii2/commit/7c9b7c77a7800ccd9d836f21c542b9828c2aa52c

**Comment by samdark** [comment]
Thanks!

**Pull request closed without merging**